lusSTR/scripts/filter_settings.py

```diff
@@ -11,15 +11,15 @@
 # -------------------------------------------------------------------------------------------------
 
 from collections import defaultdict
+import importlib_resources
 import json
 import numpy as np
 import pandas as pd
-from pkg_resources import resource_filename
 import re
 
 
 def get_filter_metadata_file():
-    return resource_filename("lusSTR", "data/filters.json")
+    return importlib_resources.files("lusSTR") / "data/filters.json"
```

Comment:
Not sure it's worth going back at this point, but if you're always using this `Path` object as a string, you could wrap it with `str` here instead of every single place you use it in the test suite.

Reply:
Here for in ```__init__.py```?

Reply:
Oops, wrong function. I meant to comment on the `data_file` function in `test/__init__.py`.

Reply:
I think I'll leave it as is for now. Maybe in the future will change it.